modules/virt-template-fields-web.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_templates/virt-creating-vm-template.adoc
+
+:_content-type: PROCEDURE
+[id="virt-template-fields-web_{context}"]
+= Custom template fields
+
+The following table outlines the editable fields when creating a new template in the {VirtProductName} web console.
```

Comment:
Clarity/localization suggestion: 

s/outlines the editable fields/lists the fields that you can edit

I'd also add a colon at the end of the sentence instead of a period.